.gitignore

```diff
@@ -9,3 +9,4 @@ build
 /.gradle
 /RUNNING_PID
 *.iml
+.envrc
```

Comment:
👀

Reply:
Meh, OK. I'll move it out to my global ignores.